### ISSUE TITLE
DO100011 - Vpax cannot be optimized [Incompatible license]

### ISSUE THREAD

**Issue by rmss-tech** [tabulartools/dax-optimizer#9]
### Describe the problem

Error 400: Vpax cannot be optimized (DO100011)
The Vpax cannot be optimized [Incompatible license].

### Power BI

September 2023 v2.121.644.0 64-bit

### Steps to reproduce the behavior

1) Exported VPAX from Tabular Editor and DAX Studio (tried both)
2) uploaded Model into DAX Optimizer
3) Clicked Analyze and received error almost immediately
ERROR:
Vpax cannot be optimized (DO100011)
The Vpax cannot be optimized [Incompatible license].
POST /v1/job/251707043214-f18d4de0-5b93-4eb8-bc83-5a5cc3edd55b/251707042078-404a9be4-06b5-4272-85c0-c702115dded1/251707041697-51212b14-ace9-410f-904d-f2d1cbf02b04
Trace id: 98c9448b-0b59-aefd-bfcc-9d1f7af083f0

### Additional context

VPAX generated from dataset on PBI Service (Premium Workspace)

### Anything else ?

Trace id: 98c9448b-0b59-aefd-bfcc-9d1f7af083f0

**Comment by albertospelta** [comment]
Hi @rmss-tech, thanks for reporting. We're investigating this and will update this issue.

**Comment by albertospelta** [comment]
@rmss-tech you are experiencing the `Vpax cannot be optimized [Incompatible license]` message because your VPAX file was generated from a PBI Service dataset but you are currently using a [DaxOptimizer DESKTOP license](https://www.tabulartools.com/p/dax-optimizer/) that can only analyze models created with Power BI Desktop.

**Comment by rmss-tech** [comment]
FYI - I was able to transfer my PBIX to another machine where I was able to install and use the latest version of DAX Studio to generate another VPAX file.  That file was able to be successfully analyzed in DAX Optimizer. Rob-----sent from my iPhoneOn Sep 21, 2023, at 1:48 AM, Alberto Spelta ***@***.***> wrote:﻿
Hi @rmss-tech, thanks for reporting. We're investigating this and will update this issue.
